### PR TITLE
Buttons quest v1

### LINKED ISSUE #16: Place buttons vertically for `/about_us`
Connected with #8 
Test how it will be more comfortable for users to have buttons

### LINKED ISSUE #17: Add inline buttons for quest questions
## Description
Following and basing on a code of #12 and #13 re-write the code, so it will be possible for users to give answers via buttons under questions. 
### Step-by-step (please mark done tasks)
- [ ]  add buttons under question with the number of an answer from the list
- [ ]  handle chosen answers from users
- [ ]  check current logic of collecting answers:
     - if logic of saving user's answers is bad then rewrite it with your own
     - if it is good then implement your code in a way that answers from your buttons will be counted by @egor-cpu code

Document necessary code changes to implement our updated code. Provide code with proper commit descriptions

### Example
image

### PR DESCRIPTION
**Pull request by NikitaZavorin**
Add inline buttons for quest questions

### CODE REVIEW

File: bot/buttons.py [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+from aiogram.fsm.context import FSMContext
+from aiogram.fsm.state import State, StatesGroup
 from aiogram.types import KeyboardButton, Message, ReplyKeyboardMarkup, ReplyKeyboardRemove
-from aiogram.filters import Command
+from aiogram.filters import Command, StateFilter
 from aiogram.types import BotCommand
 from bot import bot, dp
 
+class AboutUsStates(StatesGroup) :
+    WaitingForDepartmentChoice = State()
```
Comment (wak3upalex) [inline bot/buttons.py:10]: As far as I understand, this is how you fixed a bug with a possibility to get info from `/about_us` command?

File: bot/inLineButtons_quest.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    },
+    {
+        "question": "Какую задачу вы бы предпочли?",
+        "options": ["Решить проблему с кодом", "Написать статью", "Провести переговоры", "Провести собеседование"]
+    },
+    {
+        "question": "Какую область вы хотите развивать?",
+        "options": ["IT", "PR", "CR", "HR"]
+    }
+]
```
Comment (wak3upalex) [inline bot/inLineButtons_quest.py:37]: Any future **NEW** questions can be easily added here?

**Review by wak3upalex — COMMENTED** [review]
Please respond on comments

File: bot/inLineButtons_quest.py (hunk 2 above)
Comment (NikitaZavorin) [reply bot/inLineButtons_quest.py:37]: Yes

File: bot/buttons.py (hunk 1 above)
Comment (NikitaZavorin) [reply bot/buttons.py:10]: it is necessary for handling answers of `/about_us` only during State of this command

**Review by wak3upalex — APPROVED** [review]
Everything good and fine

**Comment by wak3upalex** [comment]
V1 of Quest/Test finally done. Good work @NikitaZavorin @egor-cpu !!!

**Pull request merged**